rosidl_generator_dotnet/cmake/rosidl_generator_dotnet_generate_interfaces.cmake

```diff
@@ -35,6 +35,15 @@ set(_output_path
 set(_generated_cs_files "")
 set(_generated_c_ts_files "")
 set(_generated_h_files "")
+set(ros2_distro "$ENV{ROS_DISTRO}")
+set(humble_distro_and_newer humble iron rolling)
+
+
+if(ros2_distro IN_LIST humble_distro_and_newer) 
+set(PYTHON_COMMAND ${PYTHON_EXECUTABLE})
+else
+set(PYTHON_COMMAND Python3::Interpreter)
```

Comment:
I get a parse error for this line on both `foxy` and `humble` on my machine (two different vm's for both ubuntu versions):

```
~/humble/ros2_dotnet_ws$ cat log/latest_build/unique_identifier_msgs/stdout_stderr.log
-- The C compiler identification is GNU 11.3.0
-- The CXX compiler identification is GNU 11.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found ament_cmake: 1.3.4 (/opt/ros/humble/share/ament_cmake/cmake)
-- Found Python3: /usr/bin/python3.10 (found version "3.10.6") found components: Interpreter
-- Found rosidl_default_generators: 1.2.0 (/home/stefan/humble/ros2_dotnet_ws/install/rosidl_default_generators/share/rosidl_default_generators/cmake)
-- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
-- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
-- Found ament_cmake_ros: 0.10.0 (/opt/ros/humble/share/ament_cmake_ros/cmake)
-- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
-- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
CMake Error at /home/stefan/humble/ros2_dotnet_ws/install/rosidl_generator_dotnet/share/rosidl_generator_dotnet/cmake/rosidl_generator_dotnet_generate_interfaces.cmake:45:
  Parse error.  Expected "(", got newline with text "

  ".
Call Stack (most recent call first):
  /opt/ros/humble/share/ament_cmake_core/cmake/core/ament_execute_extensions.cmake:48 (include)
  /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:286 (ament_execute_extensions)
  CMakeLists.txt:21 (rosidl_generate_interfaces)
```

on my humble vm i use `cmake version 3.22.1`, on foxy `cmake version 3.16.3`

Why did you change this? Do you have any clue why this would break on my machine but not on CI?
Can look at this again later on, but maybe you already know something and can give some pointers :)

Reply:
i forgot the () at line 44, was else but needed to be else()